### ISSUE TITLE
Vitest setupFiles not detecting package name from long path

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [webpro/knip#219]
Looks like Knip isn't detecting that `console-fail-test` is a dev dependency here?

```ts
// vitest.config.ts
import { defineConfig } from "vitest/config";

export default defineConfig({
  test: {
    setupFiles: ["console-fail-test/setup"],
  },
});
```

```plaintext
> @ knip /Users/josh/repos/repros
> knip

Unused dependencies (1)
console-fail-test  package.json
Unresolved imports (1)
/Users/josh/repos/repros/console-fail-test/setup  vitest.config.ts
```

Branch with full repro: https://github.com/JoshuaKGoldberg/repros/tree/knip-vitest-console-fail-test-setup

**Comment by webpro** [comment]
True. Knip expects file paths here, didn't know `setupFiles` could be dependency refs as well. People don't always start relative file paths with a `.` so that's why I "fixed" that. Apparently dependency refs are allowed so, yeah, will fix.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.20.1. See [Release 2.20.1](https://github.com/webpro/knip/releases/tag/2.20.1) for release notes._
